Makefile

```diff
@@ -38,4 +36,7 @@ shrinkwrap:
 	docker run -it -v ${MOUNT_DIR}\:${APP_DIR} -w / -u root ${IMAGE} \
 	    bash -c "npm shrinkwrap && cp npm-shrinkwrap.json ${APP_DIR}"
 
-.PHONY: run test test-macros lint lint-macros bash shrinkwrap
+src/parser.js: src/parser.pegjs
+	./node_modules/.bin/pegjs src/parser.pegjs
```

Comment:
👍 I think it was an error that we weren't generating `src/parser.js` for the production images, one that's been there for a few years. In general, I don't like checking in derived files, but this one probably changes rarely enough that it won't be an issue.